releasenotes/notes/add-unittest-support-024c4664f6546c21.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Adds unittest support for test visibility only
```

Comment:
It's unclear to me what the "only" in this note is referring to. What are the other things that "unittest" might "support"?

Reply:
Apologies, my approach is to split the unittest support into test level visibility initially and full visibility (including session, module, suite and test-level visibility). Do you think splitting them might make them easier for the reviewer? Otherwise, the full PR is available [here work in progress](https://github.com/DataDog/dd-trace-py/pull/6611)